### PR TITLE
Improved search bar handling. [WIP]

### PR DESCRIPTION
**Pull request by elland**

### CODE REVIEW

**Comment by 3lvis** [comment]
What's missing here? 🤔

**Comment by elland** [comment]
Conflict resolution at least? 😅

**Comment by yuliaveres** [comment]
And please, provide description what is improved and show it in giff 😉

**Comment by yuliaveres** [comment]
And remove `WIP`

**Comment by elland** [comment]
Updated and merged back from master to resolve conflicts. 👍

File: Demo.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1236,6 +1247,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.demo.SweetUIKit;
 				PRODUCT_NAME = SweetUIKit;
 				SKIP_INSTALL = YES;
+				SWIFT_SWIFT3_OBJC_INFERENCE = Off;
```
Comment (3lvis) [inline Demo.xcodeproj/project.pbxproj:1250]: Can you change it to Default?

**Review by 3lvis — APPROVED** [review]

**Comment by elland** [comment]
👏🏽

**Pull request merged**